Makefile

```diff
@@ -101,10 +101,9 @@ db-drop:
 	dropdb --username pointfreeco pointfreeco_test || true
 	dropuser pointfreeco || true
 
-
 xcodeproj-oss: check-dependencies
```

Comment:
We can prob delete the Xcode project and module map commands entirely.

Reply:
yeah, even for playgrounds we found a way to get the packages and playgrounds to play nicely together. so down to delete whenever we want